pin-trustee.bu

```diff
@@ -0,0 +1,21 @@
+variant: fcos
```

Comment:
Why do we have this file twice? We already have it present in https://github.com/trusted-execution-clusters/investigations/blob/main/configs/remote-ign/pin-trustee.bu